### ISSUE TITLE
Naming conflict with OpenShift CLI (oc)

### ISSUE THREAD

**Issue by mattthym** [node-opcua/opcua-commander#32]
I tried installing opcua-commander with `npm install opcua-commander -g` and it failed due to a naming conflict with OpenShift CLI.

```
npm notice Run npm install -g npm@7.21.1 to update!
npm notice 
npm ERR! code EEXIST
npm ERR! path /usr/local/bin/oc
npm ERR! EEXIST: file already exists
npm ERR! File exists: /usr/local/bin/oc
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.
```

I would be fine with running opcua-commander as is and don't need the supposedly shortened symlink created in /usr/local/bin but I am unsure how to adapt the installation process. Any ideas on that?

Also I do not wish to overwrite the symlink for the Openshift CLI as I use it more often than opcua-commander.

**Comment by emeric254** [comment]
To avoid _polluting_ your system, you could just install it locally (in a the folder of your choice)

`cd some_folder`  
`npm install opcua-commander`       (without the "-g" !)


and then, from this folder, start it using 

`npx opcua-commander`

**Comment by erossignon** [comment]
fix in opcua-commander@0.14.0
